asdf/tags/wcs/wcs.py

```diff
@@ -295,3 +314,47 @@ def assert_equal(cls, old, new):
         assert old.name == new.name
         for old_frame, new_frame in zip(old.frames, new.frames):
             helpers.assert_tree_match(old_frame, new_frame)
+
```

Comment:
The reason `ICRSCoord` is now defined here is because it is the type of the new `galcen_coord` parameter in galactocentric frames. The `ICRSCoord` is not itself a frame, so it can't be defined using the same schema. It's possible that I missed another existing schema that would subsume `ICRSCoord`, in which case this new code would not be necessary. It's also possible that `ICRSCoord` should be defined elsewhere (e.g. in its own source file under `wcs`), but since it is only being used by galactocentric frames as far as I am aware, this is where I put it for now.